results/user-testing-results.md

```diff
@@ -0,0 +1,138 @@
+
+#### Different options for feedback on cell status 
+
+**Current default JupyterLab feedback for notebook cell status is exclusively visual**; the cell prompt and favicon changes rely on users being able to see the icons, especially since these changes are not surfaced to the user unless their focus happens to already be on those icons when they change. This makes knowing the status of a cell difficult to impossible for users who cannot look at the screen in full. Quick to run code cells with a cell output—like the majority of cells run during these tests— tend to make this a trivial issue, it marks a drastic experience difference for sighted and non-sighted users. Fully sighted users can quickly verify the status of a cell in two different places, go about other work while waiting, and more easily verify when a cell has completed running. 
+
+Participants who could not easily see or could not see the staus indicators at all (since they are small and relatively low contrast) relied on prior experience with notebooks, gut instinct about how long that section of code usually takes to run, or the appearance of a cell output to check a cell’s status. Having at least one non-visual option (like an audio indicator) or a visual option that would represent itself in a timely manner for assistive tech (like a notification) could provide users with the support they want. Making these options customizable would be important as well.
+
+#### Flexible command palette search/filter behavior
+
+Because several participants relied heavily on the command palette to complete tasks, the limit of its filtering arose several times. From the user end, the command palette feels like it searches only for direct matches within the command list for whatever the user types in. If a user knows exactly what they are looking for, they often have success. Participants were sometimes able to quickly and flawlessly locate a desired command. But when they were exploring JupyterLab’s command options, unable to remember the exact name of a command, made a typo, or were unfamiliar with a task many participants had to give up using the command palette in order to complete the task. Depending on the assistive tech they use, typos can also be nontrivial to detect for blind or visually impaired users. Having a fuzzy search or other typo/error support rather than only the direct filter would help the command palette continue to be the boon many participants wanted it to be.
```

Comment:
```suggestion
Because several participants relied heavily on the command palette to complete tasks, the limit of its filtering arose several times. From the user end, the command palette feels like it searches only for direct matches within the command list for whatever the user types in. If users know precisely what they are looking for, they often succeed. Participants were sometimes able to quickly and flawlessly locate a desired command. But when they were exploring JupyterLab’s command options, unable to remember the exact name of a command, made a typo, or were unfamiliar with a task, many participants had to give up using the command palette to complete the task. Depending on their assistive tech, typos can also be nontrivial to detect for blind or visually impaired users. Having a fuzzy search or other typo/error support rather than only the direct filter would help the command palette continue to be the boon many participants wanted it to be.
```